google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -600,6 +602,30 @@ def _convert_computeproject(self, computeproject):
                 data=computeproject.get_data_raw(),
                 parent=parent))
 
+    def _convert_iam_policy(self, iam_policy):
+        """Convert an IAM policy to a database object.
+
+        Args:
+            iam_policy (object): IAM policy to store.
+        """
+        # TODO: Figure out why bucket IAM policy can not be imported.
+        unsupported_iam_policy_type = ['bucket']
```

Comment:
In _convert_bucket, the resource isn't added to the resource cache, if you update _convert_bucket to look more like _convert_dataset then it will work, and you won't need to skip it here.

Reply:
I'm okay with keeping this a todo, although the change is small enough it might be nice to include it with this PR.

Reply:
Done, fixed.  :)